src/Controller/PostController.php

```diff
@@ -95,11 +98,11 @@ public function put(Post $post, Request $request): JsonResponse {
      * @param Post $post
      * @return JsonResponse
      */
-    public function delete(Post $post): JsonResponse {
+    public function delete(Post $post): JsonResponse
+    {
         $this->entityManager->remove($post);
         $this->entityManager->flush();
 
         return $this->json(204);
```

Comment:
J’imagine que 204 c’est ton code HTTP? Dans ce cas ce serait plutôt `return $this->json(null, 204);`
Sinon 204 c’est le contenu de ta réponse et ca renvoie un code 200 :)